adapters/displayio/displayio_test.go

```diff
@@ -0,0 +1,23 @@
+package displayio
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+
+	"github.com/prebid/prebid-server/v2/adapters/adapterstest"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderDisplayio,
+		config.Adapter{Endpoint: "https://prebid.display.io/?publisher=101"},
```

Comment:
should use endpoint specified in `displayio.yaml` - `https://prebid.display.io/?publisher={{.PublisherID}}`

this ensures adapter code works correctly to substitute macro

Reply:
Endpoint updated